.github/workflows/pythonpackage.yml

```diff
@@ -71,7 +71,7 @@ jobs:
     - name: Run mypy
       shell: bash
       run: |
-        poetry run mypy fgpyo tests --config=ci/mypy.ini
+        poetry run mypy fgpyo tests --config=pyproject.toml
```

Comment:
**suggestion** I don't _think_ the `--config` flag is necessary, because it sources config from `pyproject.toml` by default

```suggestion
        poetry run mypy fgpyo tests
```